### PR TITLE
Support short DOIs in cleanDOI() and update DOI copyAsURL prefix

### PR DESCRIPTION
**Pull request by bwiernik**

### CODE REVIEW

File: chrome/content/zotero/xpcom/utilities.js [hunk 1]
```diff
@@ -311,7 +311,7 @@ Zotero.Utilities = {
 			throw "cleanDOI: argument must be a string";
 		}
 
-		var doi = x.match(/10\.[0-9]{4,}\/[^\s]*[^\s\.,]/);
+		var doi = x.match(/(?:10\.[0-9]{4,}\/[^\s]*[^\s\.,])|(?:10\/[^\s]*[^\s\.,])/);
```
Comment (dstillman) [inline chrome/content/zotero/xpcom/utilities.js]: Seems like we could shorten this to `(?:10(?:\.[0-9]{4,})?\/[^\s]*[^\s\.,])`? As in, just make the `.<number>` between the `10` and the `/` optional?

Comment (bwiernik) [reply chrome/content/zotero/xpcom/utilities.js]: Good point. Made that change.

**Comment by dstillman** [comment]
Thanks!
